feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SessionPageFragment.kt

```diff
@@ -68,6 +68,11 @@ class SessionPageFragment : DaggerFragment() {
     private val bottomSheetBehavior: BottomSheetBehavior<*>
         get() = BottomSheetBehavior.from(binding.sessionsSheet)
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setupSessionsFragment(savedInstanceState)
```

Comment:
`savedInstanceState == null` check should be here rather than inside of * setupSessionsFragment* 👀